src/components/PopupWithForm.js

```diff
@@ -22,6 +23,14 @@ export default class PopupWithForm extends Popup {
     super.setEventListeners();
   }
 
+  setLoading(loading) {
+    if (loading) {
+      this._submitButton.textContent = "Saving";
```

Comment:
`Saving` should have `...` at the end according to the task (and to show loading)


```js

Saving...
```